src/test/java/com/exasol/adapter/dialects/snowflake/SnowflakeSqlDialectTest.java

```diff
@@ -51,35 +53,19 @@ void testGetMainCapabilities() {
                         AGGREGATE_GROUP_BY_TUPLE, AGGREGATE_HAVING, ORDER_BY_COLUMN, ORDER_BY_EXPRESSION, LIMIT));
     }
 
```

Comment:
Missing capability tests for literals and predicates.
In case those are not yet supported, test for empty collection.

Reply:
added tests for literals, scalar and predicate capabilities.